django_app/redbox_app/settings.py

```diff
@@ -5,15 +5,13 @@
 from pathlib import Path
 
 import environ
-import sentry_sdk
-from sentry_sdk.integrations.django import DjangoIntegration
+# import sentry_sdk
```

Comment:
Remove commented out bits here and the sentry bit at the bottom (if we want to keep it for future-use then move the sentry stuff outside the local setup)

Reply:
[59c7228](https://github.com/i-dot-ai/redbox-copilot/pull/292/commits/59c72286dbad47ebfc927e09b39c7ca7310d1805)